azuredevops/internal/acceptancetests/resource_serviceendpoint_servicefabric_test.go

```diff
@@ -0,0 +1,59 @@
+// +build all resource_serviceendpoint_service_fabric
+// +build !exclude_serviceendpoints
+
+package acceptancetests
+
+import (
```

Comment:
`azure_active_directory` not covered by AccTests. 
Tests should cover different  scenario like, `basic create`, `update`, `complete  create`